### ISSUE TITLE
How to have a tooltip on the header that uses aria-label

### ISSUE THREAD

**Issue by hometownnerd** [Mottie/tablesorter#1011]
I have been trying to find out how to set the tooltip equal to the aria-label, even just simply using the title of the th would be fine, but I cannot get it to do that. I know I am missing something somewhere.

**Comment by Mottie** [comment]
Hi @hometownnerd!

Check out the [`headerTitles` widget](http://mottie.github.io/tablesorter/docs/example-widget-header-titles.html) which should help you out.

**Comment by hometownnerd** [comment]
Awesome! Thanks a bunch! Love this fork, by the way!!

**Comment by Mottie** [comment]
Thanks! I'm glad people are finding it useful :grin:

**Comment by hometownnerd** [comment]
one more question....I am trying to disable the sort on a leading and trailing header row....I have tried setting the  class = "sorter-false" and tried to remove the tablesorter-header class and it still wants to put the arrows on those rows. How can I get the rows to not show the arrows?

**Comment by Mottie** [comment]
When setting `class="sorter-false"`, it is actually the lowest priority setting - check out [this demo](http://mottie.github.io/tablesorter/docs/example-options-headers.html). So what that means is if you have a `headers` setting that sets a parser for that column, it will override that class.

If you shared the HTML of the `thead` and the initialization code, I might be able to help find the problem.

Also, don't forget you can always use the [`debug` option](http://mottie.github.io/tablesorter/docs/index.html#debug) to see what is happening when tablesorter initializes.

```js
$('table').tablesorter({
  // other options...
  debug: true
});

**Comment by hometownnerd** [comment]
This is my thead....
<thead>
		<tr style="color:White;background-color:#005480;" role="row" class="tablesorter-headerRow">
			<td class="sorter-false tablesorter-headerUnSorted" colspan="12" style="font-weight: bold; -webkit-user-select: none;" data-column="0" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Accounts: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Accounts</div></td>
		</tr><tr style="color:White;background-color:#005480;" role="row" class="tablesorter-headerRow">
			<td style="font-weight: bold; -webkit-user-select: none;" data-column="0" class="tablesorter-header tablesorter-headerAsc" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="ascending" aria-label=": sorting from a to z, click to sort from z to a"><div class="tablesorter-header-inner"></div></td><th style="font-weight: bold; -webkit-user-select: none;" data-column="1" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Acct No: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Acct No</div></th><th style="font-weight: bold; -webkit-user-select: none;" data-column="2" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Name: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Name</div></th><th style="font-weight: bold; -webkit-user-select: none;" data-column="3" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Plan Code: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Plan Code</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="4" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Dsch Date: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Dsch Date</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="5" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="FBill Date: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">FBill Date</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="6" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Total Chgs: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Total <br>Chgs</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="7" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Total Pmt: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Total <br>Pmt</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="8" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Total Adj: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Total <br>Adj</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="9" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Balance: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Balance</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="10" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Status: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Status</div></th><th align="right" style="font-weight: bold; -webkit-user-select: none;" data-column="11" class="tablesorter-header tablesorter-headerUnSorted" tabindex="0" scope="col" role="columnheader" aria-disabled="false" aria-controls="ctl00_ContentPlaceHolder1_tblAccts" unselectable="on" aria-sort="none" aria-label="Cases: not sorted, but click to sort from a to z"><div class="tablesorter-header-inner">Cases</div></th>
		</tr><tr style="color:White;background-color:#005480;" role="row" class="tablesorter-headerRow">
			<td class="sorter-false tablesorter-header tablesorter-headerUnSorted" align="center" colspan="12" style="font-weight: bold; -webkit-user-select: none;" data-column="0" tabindex="0" scope="col" role="columnheader" aria-disabled="true" unselectable="on" aria-sort="none"><div class="tablesorter-header-inner">Your search will return more than 200 matches, only showing 200 matches, please narrow your search parameters</div></td>
		</tr>
	</thead>
The row that has "accounts" and the row that has "Your search will return more than 200" need to not be able to cause the sort. I will turn on debug to see what is happening, but if you can find it that would be great. I am getting the tooltips working now.
Thanks again!

**Comment by Mottie** [comment]
I'm not sure why the "sorter-false" isn't working in this case, but since those two rows are separate from the others, you can add a "tablesorter-ignoreRow" class to the `<tr>` (see [`cssIgnoreRow`](http://mottie.github.io/tablesorter/docs/index.html#cssignorerow)).

**Comment by hometownnerd** [comment]
Awesome! That is perfect! thanks a bunch!

**Comment by Mottie** [comment]
Actually, maybe it is because the `headers` option has a parser set... like in [this example](http://jsfiddle.net/Mottie/856bzzeL/641/)
```js
$(function () {
	$('table').tablesorter({
		theme: 'blue',
        headers : {
            // this overrides the "sorter-false" in the first row
            0 : { sorter: 'text' }
        }
	});
});
```

**Comment by Mottie** [comment]
No problem, glad to help.

**Comment by hometownnerd** [comment]
Now I am feeling like a bit of an idiot, I am not getting the headers to work. If I have it figured out right it should populate tooltip into the tableheaders on its own, correct? It seems like it is not doing that, but on top of that the tooltip is not showing up even when I manually add that class. I am not getting any console errors and it seems like the widget is initializing properly. Argh, sorry to bother you again.

**Comment by hometownnerd** [comment]
Nevermind....I was being an idiot...I needed the tipsy css file. Sorry if I bothered you.

Have a great day!

**Comment by Mottie** [comment]
I'm guessing this issue has been resolved, so I'm going to close it. If you continue to have problems, please feel free to continue the discussion in this thread.
